### PR TITLE
Adds janitor ERT

### PR DESCRIPTION
**Pull request by VileBeggar**
## Changelog
:cl: Vile Beggar
add: CC now has access to an emergency sanitation response team.
/:cl:

## About The Pull Request
Adds a janitor ERT that admins can use when the station gets a bit too messy.
It should work but given that I don't know how to use DM correctly there are probably some errors.

image

## Why It's Good For The Game
I've seen admins use janitor ERTs sometimes before, and it's a shame there isn't already a preset for it, hence this PR.

### CODE REVIEW

**Comment by Tlaltecuhtli** [comment]
give them a nuke op chemical sprayer with acid + space cleaner or a cool backtank with that kind of cool cleaning water

File: code/game/objects/items/storage/belt.dm [hunk 1]
```diff
@@ -519,6 +519,14 @@
 		/obj/item/assembly/mousetrap
 		))
 
+/obj/item/storage/belt/janitor/full
```
Comment (Cyberboss) [inline code/game/objects/items/storage/belt.dm]:
```suggestion
```

File: code/modules/clothing/outfits/ert.dm [hunk 2]
```diff
@@ -159,6 +159,32 @@
...
+	r_pocket = /obj/item/grenade/chem_grenade/cleaner
+	l_pocket = /obj/item/grenade/chem_grenade/cleaner
+	backpack_contents = list(/obj/item/storage/box/engineer=1,\
+		/obj/item/storage/box/lights/mixed=1,\
+		/obj/item/melee/baton/loaded=1,\
+		/obj/item/clothing/mask/gas/sechailer=1,\
+		/obj/item/mop/advanced=1,\
+		/obj/item/reagent_containers/glass/bucket=1,\
+		/obj/item/grenade/clusterbuster/cleaner=3)
+/datum/outfit/ert/janitor/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```
Comment (Cyberboss) [inline code/modules/clothing/outfits/ert.dm]:
```suggestion

/datum/outfit/ert/janitor/post_equip(mob/living/carbon/human/H, visualsOnly = FALSE)
```

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by FuryMcFlurry** [comment]
post sprites

**Review by Cyberboss — DISMISSED** [review]

**Comment by Cyberboss** [comment]
@FuryMcFlurry IconDiffBot yo: https://github.com/tgstation/tgstation/pull/42517/checks?check_run_id=54834429

**Comment by FuryMcFlurry** [comment]
Ah, you have a bot that does that, neato.

**Comment by VileBeggar** [comment]
i thought the bot was broken since it didn't seem to post on prs anymore

**Comment by OverlyGeneric** [comment]
We Viscera Cleanup Detail now.

**Comment by optimumtact** [comment]
Yeah just FYI both mapdiffbot and icondiffbot are now checks that run as actual github apps thanks to work @Cyberboss has done.

also downstreams and other repos can use them which is pretty neat ~~and lets us extend our power and control over them~~

**Comment by doublepointstation** [comment]
speedmerge this

**Comment by VileBeggar** [comment]
i spent 6 hours doing something please ignore the accidental commit

**Comment by plapatin** [comment]
wait, we didn't have this before?

**Comment by optimumtact** [comment]
Did you delete the suits dmi somehow?

**Comment by VileBeggar** [comment]
@optimumtact no i didn't, it worked when i tested it

**Review by chesse20 — APPROVED** [review]
Looks Good

**Comment by KomradeSpectre** [comment]
Solid Idea but the sprite color scheme is p-bad. I suggest a lighter shade of purple ala the Janitor's outfit.

**Comment by VileBeggar** [comment]
> 
> 
> Solid Idea but the sprite color scheme is p-bad. I suggest a lighter shade of purple ala the Janitor's outfit.

guess i could make it a little bit nicer, hold on

**Comment by optimumtact** [comment]
you *must* git merge to fix icon conflicts https://github.com/tgstation/tgstation/wiki/Resolving-icon-conflicts-for-fun-and-profit

**Pull request closed without merging**